runtime/interpreter/value.go

```diff
@@ -15157,6 +15161,51 @@ func (v *CompositeValue) GetAttachments(interpreter *Interpreter, locationRange
 	return attachments
 }
 
+func (v *CompositeValue) forEachAttachmentFunction(interpreter *Interpreter, baseAuthorization Authorization, locationRange LocationRange) Value {
```

Comment:
`baseAuthorization` seems to be unused here. Is it needed for any follow-up PRs?

Reply:
Nope, that's a leftover from before when entitlements had runtime behavior. Wish Go's unused variable lints also applied to named function parameters.